content/blog/announcing_kubeflow_0.2.md

```diff
@@ -0,0 +1,109 @@
+  - [Identity Aware Proxy](https://github.com/kubeflow/kubeflow/blob/master/docs/gke/iap.md), which enables using security your services with identities, rather than VPNs and Firewalls
+  - [Katib](https://github.com/kubeflow/katib) for hyperparameter tuning
+  - [Kubernetes volume controller](https://github.com/kubeflow/experimental-kvc) which provides basic volume and data management using volumes and volume sources in a Kubernetes cluster.
+  - [Kubebench](https://github.com/kubeflow/kubebench ) for benchmarking of HW and ML stacks
+  - [Pachyderm](https://github.com/kubeflow/kubeflow/tree/master/kubeflow/pachyderm) for managing complex data pipelines
+  - [PyTorch operator](https://github.com/kubeflow/pytorch-operator) for running PyTorch jobs
+  - [Seldon Core](https://github.com/kubeflow/kubeflow/tree/master/kubeflow/seldon) for running complex model deployments and non-TensorFlow serving
+
+To everyone who has contributed so far, we would like to offer a huge thank you! We are getting ever closer to realizing our vision: letting data scientists and software engineers focus on the things they do well by giving them an easy-to-use, portable and scalable ML stack.
+
+## Introducing Kubeflow 0.2
+We’re proud to announce the availability of Kubeflow 0.2. The new release offers significant performance upgrades, a simplified getting started experience, and alpha support for advanced features. Let’s walk through some of the highlights:
+
+### Improved Getting Started Experience
+We know that many data scientists and ML engineers are new to Kubernetes and Kubernetes concepts. We’ve done a number of investments to minimize the amount of work they have to do to get started on the platform. This includes:
+
+  - A new deployment script which makes getting started on an existing cluster a single command. To get a default deployment running on Kubernetes *anywhere* just execute:
+    
+    ```
+    export KUBEFLOW_VERSION=0.2.2
+    curl https://raw.githubusercontent.com/kubeflow/kubeflow/v${KUBEFLOW_VERSION}/scripts/deploy.sh | bash
+    ```
+
+  - We also offer cloud-specific versions of these deployment scripts so that you can auto create a cluster if you don’t have one available. For example:
+    ```
+    export KUBEFLOW_VERSION=0.2.2
+    curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_VERSION}/scripts/gke/deploy.sh | bash
+    ```
+  - A central UI that gives you visibility into all the components running in your cluster - Central UI - To make it easier to navigate among components
+
+    ![Central UI Dashboard](../Central_UI_Screenshot.svg)
+
+  - A declarative deployment process that first creates Cloud resources including your Kubernetes cluster and then deploys Kubeflow. We have an example for GCP using Deployment Manager, and plan to add support for other clouds using tools like Terraform.

```

Comment:
The bullet points don't render correctly.
https://screenshot.googleplex.com/f3LgLkdvdRb.png

This is an annoying thing about Netify markdown. Try removing the blank lines before/after the image link

Reply:
The bullet point rendering is an issue caused due to something in the CSS. We had noticed this earlier. Any bullet entry that happens to be longer than a single line ends up messing the placement of the bullet itself. We'll need to work with our designer to fix this. For example, see the first bullet point in the same list.

Reply:
For eg. if I remove that empty line before the image, here is how it looks: https://screenshot.googleplex.com/ixV42QBf8d3